src/yio-interface/entities/mediaplayerinterface.h

```diff
@@ -93,13 +105,25 @@ class MediaPlayerDef : public QObject {
         C_VOLUME_SET,
         C_VOLUME_UP,
         C_VOLUME_DOWN,
+        C_CURSOR_UP,
+        C_CURSOR_DOWN,
+        C_CURSOR_LEFT,
+        C_CURSOR_RIGHT,
+        C_CURSOR_OK,
```

Comment:
I'd prefer `C_OK` or `C_SELECT` instead.
What do you think @martonborzak ? Should we just remove CURSOR altogether for all new navigation commands? I think the name `cursor` should be avoided.

Reply:
we can use DPAD instead to have a clear connection to the hardware buttons?

Reply:
Yeah, let's do that. But if we remove it here, we need to scan through the code in `remote-software`.

Reply:
Done